pharmd-app/src/components/Fields/NoteField.js

```diff
@@ -35,7 +34,7 @@ const Title = styled.h3`
   height: auto;
 `;
 
-const Date = styled.h4`
+const DateLabel = styled.h4`
```

Comment:
Had to change the naming of this because I needed to use the actual `Date` constructor